pkg/workflow/def/workflow/fork_join.go

```diff
@@ -0,0 +1,42 @@
+package workflow
+
+import "github.com/conductor-sdk/conductor-go/pkg/http_model"
+
+type ForkTask struct {
+	Task
+	forkedTasks [][]TaskInterface
+}
+
+func Fork(taskRefName string, tasks ...[]TaskInterface) *ForkTask {
+	return &ForkTask{
+		Task: Task{
+			name:              taskRefName,
+			taskReferenceName: taskRefName,
+			description:       "",
+			taskType:          FORK_JOIN,
+			optional:          false,
+			inputParameters:   map[string]interface{}{},
+		},
+		forkedTasks: tasks,
+	}
+}
+
+func (task *ForkTask) toWorkflowTask() []http_model.WorkflowTask {
+	forkWorkflowTask := task.Task.toWorkflowTask()[0]
```

Comment:
should we check the length before accessing to the first element?

Reply:
toWorkflowTask in the Task always returns an array with 1 element, so no need.